connecting-to-testnet/index.md

```diff
@@ -6,40 +6,16 @@ order: 0
 
-- Get and install the software
-- Start the Node
-- Use the Cosmos SDK CLI
-- Connect to the public endpoints
-
-The [Cosmos Network](https://cosmos.network) is an expanding ecosystem of connected blockchains. Many blockchains in the ecosystem are built with [Tendermint Core](https://docs.tendermint.com/), [Cosmos SDK](https://docs.cosmos.network/) and connected with [IBC](https://ibcprotocol.org/). One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019 and you can learn how to [join the network as a validator](https://hub.cosmos.network/main/validators/validator-setup.html). In this guide you'll learn how to start a full-node for and interface with a Cosmos Hub **testnet**.
+One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019. You can learn how to join the network as a validator, see [Run a Validator on the Cosmos Hub Mainnet](https://hub.cosmos.network/main/validators/validator-setup.html). 
 
 ## What is a testnet?
 
 A testnet is a network launched for developers and validators to test the functionality of the blockchain before the main network is launched. The state of a testnet can be reset at any time.
 
-## Public Endpoints
-
-When you have an application that needs access to the testnet, there is an infrastructure available that you can use.
-Connect to the Cosmos Hub testnet using the following endpoints:
-
-| Service      | URL                                                                                    |
-| ------------ | -------------------------------------------------------------------------------------- |
-| RPC          | [https://rpc.testnet.cosmos.network:443](https://rpc.testnet.cosmos.network:443)       |
-| API          | [https://api.testnet.cosmos.network:443](https://api.testnet.cosmos.network:443)       |
-| gRPC         | [https://grpc.testnet.cosmos.network:443](https://grpc.testnet.cosmos.network:443)     |
-| Token faucet | [https://faucet.testnet.cosmos.network:443](https://faucet.testnet.cosmos.network:443) |
-
-## Seed Nodes
-
-* `30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819`
-* `754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654`
-
-## Requesting Tokens from a Faucet
+## Tutorial Testnet 
 
-The faucet has a simple [web-interface](https://faucet.testnet.cosmos.network), but you can also request tokens programmatically:
+The source code repo `b-harvest/gravity-dex` offers more functionality because it adds the `tendermint/liquidity` module that is the source code of Gravity DEX to the `cosmos/gaia` source code behind Cosmos Hub. 
 
-```
-curl -X POST -d '{"address": "cosmos1kd63kkhtswlh5vcx5nd26fjmr9av74yd4sf8ve"}' https://faucet.testnet.cosmos.network
-```
\ No newline at end of file
+Now that you know what a testnet is, it's time to get started. See [About the Testnet Tutorial](testnet-tutorial.md).
```

Comment:
@fadeev what's our recommended link syntax for files at the same level? both ./filename.md and filename.md seem to work) magic?

Reply:
Let's be explicit and use `.md`.